src/input/src/interactions/pairs.rs

```diff
@@ -206,6 +206,8 @@ fn read_pair_potential(pair: &Table) -> Result<Box<PairPotential>> {
             "lj" => Ok(Box::new(try!(LennardJones::from_toml(table)))),
             "buckingham" => Ok(Box::new(try!(Buckingham::from_toml(table)))),
             "born" => Ok(Box::new(try!(BornMayerHuggins::from_toml(table)))),
+            "MorsePotential" => Ok(Box::new(try!(MorsePotential::from_toml(table)))),
```

Comment:
Since we are using short names as keys here, maybe naming it `morse` is more suitable here?

This will be the name by which the user can define the potential in the toml file, i.e.
```toml
[[pairs]]
atoms = ["A", "B"]
morse = {depth= ...
```